policy/predicate/status.go

```diff
@@ -27,10 +27,17 @@ type HasSuccessfulStatus []string
 
 var _ Predicate = HasSuccessfulStatus([]string{})
 
-func (pred HasSuccessfulStatus) Evaluate(ctx context.Context, prctx pull.Context) (bool, string, error) {
+func (pred HasSuccessfulStatus) Evaluate(ctx context.Context, prctx pull.Context) (*common.PredicateResult, error) {
 	statuses, err := prctx.LatestStatuses()
+
+	predicateResult := common.PredicateResult{
+		Satisfied:       false,
+		ValuePhrase:     "check",
+		ConditionPhrase: "has successful status",
```

Comment:
```suggestion
		ValuePhrase:     "status checks",
		ConditionPhrase: "exist and pass",
```

Here's another place that doesn't set `ConditionMap` or `ConditionValues`